### ISSUE TITLE
Custom posters for presentations

### ISSUE THREAD

**Issue by pavelsokolov** [dsv-su/play#59]
It should be possible to upload a custom poster for presentations. Right now it's only available on the upload view, should be added to the edit view as well.

**Comment by orkit** [comment]
Yes, that would be a good enhancement. For this we need a new endpoint from the play-store api for this purpose.
It would also be good if we design the entire editing process, the workflow, for example if we only want to be able to update a poster or if it should also be possible to update the entire presentation and in that case it should be done individually or all at once.
